app/src/main/java/com/example/tiptime/MainActivity.kt

```diff
@@ -89,6 +111,23 @@ private fun calculateTip(amount: Double, tipPercent: Double = 15.0): String {
     return NumberFormat.getCurrencyInstance().format(tip)
 }
 
+@Composable
+fun EditNumberField(
+        value: String,
+        onValueChange: (String) -> Unit,
+        modifier: Modifier = Modifier) {
+    OutlinedTextField(
+        value = value,
+        onValueChange = onValueChange,
+        modifier = modifier,
+        label = {Text(stringResource(R.string.bill_amount))},
```

Comment:
Форматування, лямбди зазвичай розкривають.
```suggestion
        label = {
            Text(stringResource(R.string.bill_amount))
        },
```

 Бувають випадки коли можна в одному рядку, наприклад `.filter { it == 10 }` тут мало символів в дужках